vis/test/store/zoom.test.js

```diff
@@ -32,39 +33,84 @@ describe("zoom state", () => {
   });
 
   describe("reducers", () => {
-    it("should return the initial state", () => {
-      const expectedResult = false;
+    describe("zoom reducer", () => {
+      it("should return the initial state", () => {
+        const expectedResult = false;
 
-      const result = reducer(undefined, {});
+        const result = zoomReducer(undefined, {});
 
-      expect(result).toEqual(expectedResult);
-    });
+        expect(result).toEqual(expectedResult);
+      });
 
-    it("should handle the zoom in", () => {
-      const initialState = false;
-      const expectedResult = true;
+      it("should handle the zoom in", () => {
+        const initialState = false;
+        const expectedResult = true;
 
-      const result = reducer(initialState, zoomInFromMediator());
+        const result = zoomReducer(initialState, zoomInFromMediator());
 
-      expect(result).toEqual(expectedResult);
-    });
+        expect(result).toEqual(expectedResult);
+      });
+
+      it("should handle the zoom out", () => {
+        const initialState = true;
+        const expectedResult = false;
+
+        const result = zoomReducer(initialState, zoomOutFromMediator());
 
-    it("should handle the zoom out", () => {
-      const initialState = true;
-      const expectedResult = false;
+        expect(result).toEqual(expectedResult);
+      });
 
-      const result = reducer(initialState, zoomOutFromMediator());
+      it("should handle the internal zoom out", () => {
+        const initialState = true;
+        const expectedResult = false;
 
-      expect(result).toEqual(expectedResult);
+        const result = zoomReducer(initialState, zoomOut());
+
+        expect(result).toEqual(expectedResult);
+      });
     });
 
-    it("should handle the internal zoom out", () => {
-      const initialState = true;
-      const expectedResult = false;
+    describe("selectedBubble reducer", () => {
```

Comment:
If the main code was stuck in separate files this could be too. Also just a style thing but might make the tests more readable (having this in a file called zoom.test.js might make it harder to find)

Reply:
Yeah, I'm actually still not sure how to organize the tests.

My idea is to have a single test file for each action type. This file has two actions though, so it's not a very good example.

An alternative is to have a single test file for each reducer. Is that what you mean? Or is there a better solution?